db/migrate/20230118203632_change_field_properties.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeFieldProperties < ActiveRecord::Migration[7.0]
+  def change
+      change_column :users, :Bio, :text
+      change_column :posts, :Text, :text
+  end
+end
```

Comment:
This is the migration that was used to make the update

Reply:
@JonahKayizzi, I see you are correct. I'm sorry for that. I don't know why the changes never feature in your pushed files. I even clone the project locally and checkout to the `create-data-model` but didn't see the new changes. 


Your project has been approved :tada:

Good luck!

Reply:
Thank you